templates/chatbots/partials/channel_dialog.html

```diff
@@ -63,3 +63,114 @@ <h3 class="font-bold text-lg">
     <button>close</button>
   </form>
 </dialog>
+{% if embed_code and widget_token %}
+    <dialog id="embedded_widget_success_modal" class="modal" x-data="{ autoShow: true }" x-init="if (autoShow) $el.showModal()">
+      <div class="modal-box max-w-4xl">
+        <h3 class="font-bold text-lg text-success mb-4">
+          <i class="fa-solid fa-check-circle me-2"></i>
+          Embedded Chat Widget Created Successfully!
+        </h3>
+        <div class="card bg-base-200 mb-4">
+          <div class="card-body">
+            <h2 class="card-title text-base">
+              <i class="fa-solid fa-key me-1"></i>
+              Widget Authentication Token
+            </h2>
+            <div class="join w-full">
+              <input type="text" class="input input-bordered join-item flex-1 font-mono text-sm"
+                     id="widget-token-copy" value="{{ widget_token }}" readonly>
+              <button class="btn btn-outline join-item" type="button"
+                      onclick="SiteJS.app.copyToClipboard(this, 'widget-token-copy')"
+                      title="Copy token to clipboard">
+                  <i class="fa-solid fa-copy"></i> Copy
+              </button>
+            </div>
+            <div class="text-sm text-warning mt-2">
+              <i class="fa-solid fa-shield-alt me-1"></i>
+              This token authenticates your widget. Keep it secure and don't share it publicly.
+            </div>
+          </div>
+        </div>
+
+        <div class="card bg-base-200 mb-4">
+          <div class="card-body">
+            <h2 class="card-title text-base">
+              <i class="fa-solid fa-globe me-1"></i>
+              Allowed Domains
+            </h2>
+            {% if allowed_domains %}
+              <ul class="list-none space-y-1">
+              {% for domain in allowed_domains %}
+                <li class="flex items-center">
+                  <i class="fa-solid fa-check text-success me-2"></i>
+                  <span class="font-mono">{{ domain }}</span>
+                </li>
+              {% endfor %}
+              </ul>
+            {% else %}
+              <div class="alert alert-warning">
+                <i class="fa-solid fa-exclamation-triangle"></i>
+                <span><strong>All domains allowed</strong> - Consider restricting to specific domains for security.</span>
+              </div>
+            {% endif %}
+            <div class="text-sm opacity-70 mt-2">
+              The widget will only work on these authorized domains.
+            </div>
+          </div>
+        </div>
+
+        <div class="card bg-base-200 mb-4">
+          <div class="card-body">
+            <div class="flex justify-between items-center mb-3">
+              <h4 class="card-title text-base mb-0">
+                <i class="fa-solid fa-code me-1"></i>
+                Embed Code
```

Comment:
I'm thinking it would be useful to show this section (the code) when I edit the channel again so I can find the code easily again with my channel specific values prepopulated

Reply:
the dialog is shown when updating the channel as well 👍